util/common/traces.go

```diff
@@ -112,3 +112,13 @@ func SegmentValidationTest(t *testing.T, traceTest TraceTestConfig, segments []t
 	return nil
 
 }
+func GenerateTraces(traceTest TraceTestConfig) error {
+	CopyFile(traceTest.AgentConfigPath, ConfigOutputPath)
```

Comment:
This never starts the agent after copying the file over.

Reply:
That function handles only trace generation, and the validator is the that actually starts the agent.  If it makes it more coherent  I can move the CopyFile there as well.